tasks/create-pyxis-image/create-pyxis-image.yaml

```
@@ -156,19 +156,66 @@ spec:
                   ORAS_ARGS+=("--platform $os/$arch")
                 fi
 
+                # Save the OCI manifest locally, to pass to a script to create the pyxis entry
+                MANIFEST_FILE="$(workspaces.data.path)/oras-manifest-fetch.json"
```

I would recommend to use the same subdirectory that we use for other stuff in a given PLR, like this:
```
PYXIS_DATA_PATH="$(dirname "$(params.snapshotPath)")/pyxis.json"
```
```suggestion
                MANIFEST_FILE="$(workspaces.data.path)/$(dirname "$(params.snapshotPath)")/oras-manifest-fetch.json"
```

---

Makes sense! Done in f9f66fc.